extendr-api/src/robj/functions.rs

```diff
@@ -1,6 +1,86 @@
-use super::*;
+use crate::*;
+
+
+/// Create a new, empty environment parented on global_env()
+/// Use the Env{} wrapper for more detail.
+/// ```
+/// use extendr_api::*;
+/// extendr_engine::start_r();
+/// let env = new_env();
+/// assert_eq!(env.len(), 0);
+/// ```
+pub fn new_env() -> Robj {
+    call!("new.env", TRUE, global_env(), 29).unwrap()
+}
+
+/// Get a global variable from global_env() and ancestors.
+///
+/// See also [global_var()].
+/// ```
+///    use extendr_api::*;
+///    extendr_engine::start_r();
+///
+///    let iris = global_var(sym!(iris)).unwrap().eval().unwrap();
+///    assert_eq!(iris.len(), 5);
+/// ```
+pub fn global_var<K : Into<Robj>>(key: K) -> Option<Robj> {
+    global_env().find_var(key)
+}
+
+/// Get a global function from global_env() and ancestors.
+/// ```
+/// use extendr_api::*;
+/// extendr_engine::start_r();
+/// let ls = global_function(sym!(ls)).unwrap();
+/// assert_eq!(ls.is_function(), true);
+/// // Note that the following will throw an R error:
+/// // let iris = global_function(sym!(iris));
```

Comment:
Can you explain in the comment why the error? Is it because `iris` is not a function?